src/lib/exports.ts

```diff
@@ -189,10 +204,108 @@ const generatePrintToPdfHTML = (
   `
 }
 
+const blobToBase64 = async (blob: Blob) => {
```

Comment:
`convertBlobToBase64`

Reply:
I prefer to use conventional functions rather than arrow functions

`function convertBlobToBase64(blob: Blob) {...}`

Reply:
Oh I see, didn't know that. Updated